### PR TITLE
Update Italian UI language file

### PR DESCRIPTION
**Pull request by bovirus**
@YerongAI 

Italian language fully revised (100% translated and strings checked in palce).
Please merge.
If you can please add my nick as translator and credits.

 Thanks.

### CODE REVIEW

File: src/OfficeToolPlus/Dictionaries/Languages/it-it.xaml [hunk 1]
```diff
@@ -80,130 +80,130 @@
...
+    <sys:String x:Key="Deploy_Mode.Create_ISO">Crea file ISO</sys:String>
     <sys:String x:Key="Deploy_Architecture">Architettura</sys:String>
-    <sys:String x:Key="Deploy_Architecture_Current_Installed">Architettura di Office attualmente installata: {0}.</sys:String>
+    <sys:String x:Key="Deploy_Architecture_Current_Installed">Architettura Office attualmente installata: {0}.</sys:String>
     <sys:String x:Key="Deploy_Architecture.x86">32 bit</sys:String>
     <sys:String x:Key="Deploy_Architecture.x64">64 bit</sys:String>
-    <sys:String x:Key="Deploy_Architecture.ARM">ARM (32-bit compatibile)</sys:String>
-    <sys:String x:Key="Deploy_Architecture.ARM64">ARM64 (64-bit compatibile)</sys:String>
+    <sys:String x:Key="Deploy_Architecture.ARM">ARM (32bit compatibile)</sys:String>
+    <sys:String x:Key="Deploy_Architecture.ARM64">ARM64 (64bit compatibile)</sys:String>
```
Comment (YerongAI) [inline src/OfficeToolPlus/Dictionaries/Languages/it-it.xaml:137]: You changed `32-bit` to `32bit`, the line 134 and 135 should be changing too? I'm not sure, please check.

In the source English, the line 134 and 135 is `32-bit` and `64-bit`.

File: src/OfficeToolPlus/Dictionaries/Languages/it-it.xaml [hunk 2]
```diff
@@ -679,188 +679,188 @@
...
-    <sys:String x:Key="Ques_Uninstall_Product_Key">Sei sicuro di voler disinstallare il codice product key?</sys:String>
-    <sys:String x:Key="Ques_Delete_vNextLicense">Sei sicuro di voler eliminare la licenza?</sys:String>
-    <sys:String x:Key="Ques_Remove_Office">Sei sicuro di dover ripulire Office? Una volta avviata, questa operazione non può essere annullata.</sys:String>
-    <sys:String x:Key="Ques_Invoke_Office_Update">Il canale di aggiornamento è stato modificato, controllare ora gli aggiornamenti di Office?</sys:String>
+    <sys:String x:Key="Ques_Install_Office.UpdateChannel">Canale aggiornamento: {0}</sys:String>
+    <sys:String x:Key="Ques_Install_Office.OfficeVersion">Versione Office: {0}</sys:String>
+    <sys:String x:Key="Ques_Install_Office.SourcePath">Sorgente file installazione: {0}</sys:String>
+    <sys:String x:Key="Ques_Deploy_Configuration">Visualizza il codice per la configurazione dettagliata, la configurazione di base qui sopra è corretta?</sys:String>
+    <sys:String x:Key="Ques_Delete_All_Installations">Vuoi  eliminare tutti i file di installazione?</sys:String>
```
Comment (YerongAI) [inline src/OfficeToolPlus/Dictionaries/Languages/it-it.xaml:774]: There are two spaces here, please check.

File: src/OfficeToolPlus/Dictionaries/Languages/it-it.xaml [hunk 3]
```diff
@@ -338,25 +338,25 @@
...
-    <sys:String x:Key="Activate_License_vNext_Metadata.RenewalTime">Prossima data di rinnovo:</sys:String>
+    <sys:String x:Key="Activate_License_vNext_Metadata.RenewalTime">Data prossima rinnovo:</sys:String>
     <!-- Legacy License -->
     <sys:String x:Key="Activate_License_Lic.Description">Descrizione:</sys:String>
     <sys:String x:Key="Activate_License_Lic.Key">Ultimi 5 caratteri della chiave:</sys:String>
     <sys:String x:Key="Activate_License_Lic.Period">Periodo rimanente:</sys:String>
     <sys:String x:Key="Activate_License_Lic.Error_Code">Codice errore:</sys:String>
     <sys:String x:Key="Activate_License_Lic.Error_Description">Descrizione errore:</sys:String>
-    <sys:String x:Key="Activate_License_VL.Type">Tipo di attivazione:</sys:String>
+    <sys:String x:Key="Activate_License_VL.Type">Tipo  attivazione:</sys:String>
```
Comment (YerongAI) [inline src/OfficeToolPlus/Dictionaries/Languages/it-it.xaml:348]: There are two spaces here, please check.

File: src/OfficeToolPlus/Dictionaries/Languages/it-it.xaml [hunk 4]
```diff
@@ -212,121 +212,121 @@
...
-    <sys:String x:Key="Deploy_Advanced_Source_Path">Percorso di origine</sys:String>
-    <sys:String x:Key="Deploy_Advanced_Source_Path.Description">Se il percorso di origine è vuoto, la rete CDN di Office viene utilizzata come origine per impostazione predefinita. In modalità download, determina dove salvare i file.</sys:String>
+    <sys:String x:Key="Deploy_Advanced_Version.Description">Se non specificato il valore predefinito è la versione più recente di Office disponibile.</sys:String>
+    <sys:String x:Key="Deploy_Advanced_Download_Path">Percorso download</sys:String>
+    <sys:String x:Key="Deploy_Advanced_Download_Path.Description">Definisce da dove vengono scaricati i file di installazione; quando si usa il percorso download è necessario specificare la versione di Office.</sys:String>
+    <sys:String x:Key="Deploy_Advanced_Source_Path">Percorso sorgente</sys:String>
+    <sys:String x:Key="Deploy_Advanced_Source_Path.Description">Se il percorso sorgente è vuoto,  per impostazione predefinita viene usata come sorgente la rete CDN di Office. In modalità download, determina dove salvare i file.</sys:String>
```
Comment (YerongAI) [inline src/OfficeToolPlus/Dictionaries/Languages/it-it.xaml:226]: There are two spaces here, please check.

File: src/OfficeToolPlus/Dictionaries/Languages/it-it.xaml [hunk 5]
```diff
@@ -80,130 +80,130 @@
...
+    <sys:String x:Key="Deploy_Module">Modulo installazione</sys:String>
+    <sys:String x:Key="Deploy_Module.Description">Per una migliore compatibilità scegli Strumento distribuzione Office Deployment Tool o per esperienze migliori lo Strumento di Office Plus.</sys:String>
+    <sys:String x:Key="Deploy_Module.ODT">Strumento distribuzione Office</sys:String>
     <sys:String x:Key="Deploy_Module.OTP">Office Tool Plus</sys:String>
-    <sys:String x:Key="Deploy_Download_First">Prima scaricalo, poi distribuiscilo</sys:String>
-    <sys:String x:Key="Deploy_Download_First.Description">I file di installazione verranno salvati nella directory del programma o nella directory specificata dal percorso di origine.</sys:String>
-    <sys:String x:Key="Deploy_Create_Shortcut">Crea un collegamento sul desktop</sys:String>
-    <sys:String x:Key="Deploy_Create_Shortcut.Description">Crea collegamenti sul desktop alle applicazioni di Office dopo l'installazione di Office.</sys:String>
+    <sys:String x:Key="Deploy_Download_First">Scarica i file  e poi distribuiscili</sys:String>
```
Comment (YerongAI) [inline src/OfficeToolPlus/Dictionaries/Languages/it-it.xaml:144]: There are two spaces here, please check.

File: src/OfficeToolPlus/Dictionaries/Languages/it-it.xaml [hunk 6]
```diff
@@ -80,130 +80,130 @@
...
+    <sys:String x:Key="Deploy_Download_First.Description">I file di installazione verranno salvati nella cartella del programma o nella cartella specificata nel percorso sorgente.</sys:String>
+    <sys:String x:Key="Deploy_Create_Shortcut">Crea collegamento sul desktop</sys:String>
+    <sys:String x:Key="Deploy_Create_Shortcut.Description">Dopo l'installazione di Office crea i collegamenti sul desktop alle applicazioni di Office.</sys:String>
     <!-- Do not edit it or translate it. iSlide only provided in mainland China. -->
-    <sys:String x:Key="Deploy_iSlide">PlaceHolder</sys:String>
-    <sys:String x:Key="Deploy_iSlide.Description">PlaceHolder</sys:String>
-    <sys:String x:Key="Deploy_iSlide_Install">PlaceHolder</sys:String>
-    <sys:String x:Key="Deploy_iSlide_Install.Name">PlaceHolder</sys:String>
+    <sys:String x:Key="Deploy_iSlide">Segnaposto</sys:String>
```
Comment (YerongAI) [inline src/OfficeToolPlus/Dictionaries/Languages/it-it.xaml:149]: There is no need to translate line 149 to 152.

File: src/OfficeToolPlus/Dictionaries/Languages/it-it.xaml [hunk 7]
```diff
@@ -494,155 +494,155 @@
...
     <sys:String x:Key="File_Type.RTF">Formato RTF</sys:String>
     <!-- Image -->
-    <sys:String x:Key="File_Type.GIF">GIF Formato Animato</sys:String>
-    <sys:String x:Key="File_Type.JPG">JPEG Formato File Interscambiabile</sys:String>
-    <sys:String x:Key="File_Type.PNG">PNG Formato Grafico Network Portabile </sys:String>
-    <sys:String x:Key="File_Type.TIF">TIFF Formato Tag File Immagine</sys:String>
-    <sys:String x:Key="File_Type.BMP">DIB Device Bitmap Indipendente</sys:String>
+    <sys:String x:Key="File_Type.GIF">GIF animato</sys:String>
+    <sys:String x:Key="File_Type.JPG">JPEG JFIF</sys:String>
+    <sys:String x:Key="File_Type.PNG">PNG </sys:String>
```
Comment (YerongAI) [inline src/OfficeToolPlus/Dictionaries/Languages/it-it.xaml:503]: Remove trailing spaces.

File: src/OfficeToolPlus/Dictionaries/Languages/it-it.xaml [hunk 8]
```diff
@@ -494,155 +494,155 @@
...
+    <sys:String x:Key="Hash_Checker_XXH128.Description">Abilita/disabilita calcolo XXH128.</sys:String>
+    <sys:String x:Key="Hash_Checker_MD5.Description">Abilita/disabilita calcolo MD5.</sys:String>
+    <sys:String x:Key="Hash_Checker_SHA1.Description">Abilita/disabilita calcolo SHA-1.</sys:String>
+    <sys:String x:Key="Hash_Checker_SHA256.Description">Abilita/disabilita calcolo SHA-256.</sys:String>
+    <sys:String x:Key="Hash_Checker_SHA384.Description">Abilita/disabilita calcolo SHA-384.</sys:String>
+    <sys:String x:Key="Hash_Checker_SHA512.Description">Abilita/disabilita calcolo SHA-512.</sys:String>
+    <sys:String x:Key="Hash_Checker_SHA3_256.Description">Abilita/disabilita elaborazione SHA3-256, supportata solo in Windows build 25324 o successiva.</sys:String>
+    <sys:String x:Key="Hash_Checker_SHA3_384.Description">Abilita/disabilita elaborazione SHA3-384, supportata solo in Windows build 25324 o successiva</sys:String>
+    <sys:String x:Key="Hash_Checker_SHA3_512.Description">Abilita/disabilita elaborazione SHA3-512, supportata solo in Windows build 25324 o successiva</sys:String>
```
Comment (YerongAI) [inline src/OfficeToolPlus/Dictionaries/Languages/it-it.xaml:562]: Line 561 and 562 missing trailing period.

**Review by YerongAI — CHANGES_REQUESTED** [review]

**Comment by YerongAI** [comment]
Also, I see you are changing `Clear list` to `Azzera elenco`.

I don't know Italian, I just want to ask if these translations are appropriate in context and requirements. Just confirm again.

File: src/OfficeToolPlus/Dictionaries/Languages/it-it.xaml (hunk 1 above)
Comment (bovirus) [reply src/OfficeToolPlus/Dictionaries/Languages/it-it.xaml:137]: Fixed (remove space in line 134 e 135)

File: src/OfficeToolPlus/Dictionaries/Languages/it-it.xaml (hunk 2 above)
Comment (bovirus) [reply src/OfficeToolPlus/Dictionaries/Languages/it-it.xaml:774]: Fixed (removed space form line 774)

File: src/OfficeToolPlus/Dictionaries/Languages/it-it.xaml (hunk 3 above)
Comment (bovirus) [reply src/OfficeToolPlus/Dictionaries/Languages/it-it.xaml:348]: Fixed (removed space form line 348)

File: src/OfficeToolPlus/Dictionaries/Languages/it-it.xaml (hunk 4 above)
Comment (bovirus) [reply src/OfficeToolPlus/Dictionaries/Languages/it-it.xaml:226]: Fixed (removed space form line 226)

File: src/OfficeToolPlus/Dictionaries/Languages/it-it.xaml (hunk 5 above)
Comment (bovirus) [reply src/OfficeToolPlus/Dictionaries/Languages/it-it.xaml:144]: Fixed (removed space form line 144)

File: src/OfficeToolPlus/Dictionaries/Languages/it-it.xaml (hunk 6 above)
Comment (bovirus) [reply src/OfficeToolPlus/Dictionaries/Languages/it-it.xaml:149]: Fixed (restored origional english strings).

File: src/OfficeToolPlus/Dictionaries/Languages/it-it.xaml (hunk 7 above)
Comment (bovirus) [reply src/OfficeToolPlus/Dictionaries/Languages/it-it.xaml:503]: Fixed (removed end space on line 503)

File: src/OfficeToolPlus/Dictionaries/Languages/it-it.xaml (hunk 8 above)
Comment (bovirus) [reply src/OfficeToolPlus/Dictionaries/Languages/it-it.xaml:562]: Fixed (Add "." at the end of lines 561/562)

**Comment by bovirus** [comment]
@YerongAI 

"Clear" iin software language is "Azzera" o"Svuota" (dpened by context.
"Cancella" is similar to "Delete" or "Remove" (it means delete something).

**Comment by bovirus** [comment]
Some proposal for GUI chnages In Deply the drop down that contain Product name seems too short and with some Office version the name is truncated on the right
Increse the drop down width
image

**Review by YerongAI — APPROVED** [review]

**Comment by bovirus** [comment]
In Toolbox -> Windows Toolbox the action opf first tjree liens are "Reset", "Rebuild" "Reste" but the button is always "Repair".
I beleiev that button could have a different meaning for each different actions
image

**Comment by bovirus** [comment]
In "Convert document" the drop down seems too short when you select longer valuee (value is showed truncated on the right)
Could increase dop down width?
image

**Comment by bovirus** [comment]
@YerongAI 

Please check if you camn fix teh UI issues.
If you like I will open a new ticket for this.

**Comment by YerongAI** [comment]
I don’t recommend adding line breaks to long sentences. It can cause unfriendly layout when the window is resized.

**Comment by bovirus** [comment]
@YerongAI 

I tested long string in place and seems are OK.

Please as I mentioned to check drop down control width.
In some case with longer stringer the value slected is truncated on the right.

If you can merge all latest changes and release a new build.

It seems that some string are not loaded right use external command.
For exlaple "Chiudi" (Close) used for upper right X remain unchganged (in my translation is Esci).
The same for "Canale corrente" (Current channel) - I translated as "Canale attuale".,

**Comment by bovirus** [comment]
I wil close the PR and I will open a new one.

**Pull request closed without merging**